cypress/e2e/1-trial-test/firstTest.cy.js

```diff
@@ -1,17 +1,39 @@
-describe('First test suite', () => {
+// describe('First test suite', () => {
```

Comment:
Pls delete this comments, you don’t need it any more.
If you want to see them, by chance, you always have git history